#goal definition
float32[] position
float32[] rate
float32 time
uint32 eeindex
uint32 home
float32 gripper_value
float32 gripper_rate
---
#result definition
float32[] position
---
#feedback
float32[] position
float32[] rate
float32 time
float32 gripper_feed_value
float32 gripper_feed_rate
